mobilecoind/README.md

```diff
@@ -1,10 +1,10 @@
-## The MobileCoin Daemon
+## mobilecoind
 
-`mobilecoind` is a standalone executable, which provides blockchain synchronization and wallet services.
+The `mobilecoind` daemon is a standalone executable, which provides blockchain synchronization and wallet services.
 
-It creates encrypted, attested connections to consensus validators who are participating in federated voting in order to get the current block height, block headers, and to submit transactions. These consensus validators are considered highly trusted due to their use of SGX, and they are used as a reliable source for block information as well as to validate and process the proposed transactions from `mobilecoind`.
+It creates encrypted, attested connections to validator nodes who are participating in federated voting in order to get the current block height, block headers, and to submit transactions. These validator nodes are considered highly trusted due to their use of SGX, and they are used as a reliable source for block information as well as to validate and process the proposed transactions from `mobilecoind`.
 
-To keep the blockchain in sync, the MobileCoin Daemon downloads new blocks from cloud storage and checks with the consensus validators that the block headers are correct. If this succeeds, they are added to a local copy of the blockchain called the Ledger DB. This is done periodically and ensures the local copy of the blockchain is fresh enough to calculate balances and generate transactions.
+To keep the blockchain in sync, the MobileCoin Daemon downloads new blocks from cloud storage and checks with the validator nodes that the block headers are correct. If this succeeds, they are added to a local copy of the blockchain called the Ledger DB. This is done periodically and ensures the local copy of the blockchain is fresh enough to calculate balances and generate transactions.
```

Comment:
if we decide to use mobilecoind, let's be consistent here? we don't introduce MobileCoin Daemon as a term above anymore.